src/create_disk.sh

```diff
@@ -467,6 +467,15 @@ chroot_run() {
     done
 }
 
+generate_gpgkeys() {
+    tmp_home=$(mktemp -d /tmp/gpg-XXXXXX)
+    gpg --homedir "${tmp_home}" --batch --passphrase '' --yes --quick-gen-key secex default
+    gpg --homedir "${tmp_home}" --armor --export secex > "/srv/builds/latest/s390x/ignition.gpg.pub"
+    gpg --homedir "${tmp_home}" --armor --export-secret-key secex > "/tmp/ignition.asc"
+    touch "$deploy_root/etc/ignition.asc"
```

Comment:
Let's move this to where we bind mount over it.

Reply:
>Let's move this

later we `mount -o remount,ro /` and calculate the hash, so cannot move

Reply:
workaroud: key is attached to ramdisk as `/tmp/ignition.asc`